seqseek/lib.py

```diff
@@ -152,6 +152,9 @@
     'NT_167251.1': 1680828,
 }
 
+MITOCHONDRIA_NAMES = ('MT', 'RSRS', BUILD37_ACCESSIONS['MT'], BUILD37_ACCESSIONS['RSRS'],
+                      BUILD38_ACCESSIONS['MT'], BUILD38_ACCESSIONS['RSRS'])
```

Comment:
can this be a set for both performance and de-duplication since the accessions are the same on both 37 & 38?

Reply:
Done